### PR TITLE
Android release notes for `1.40.x - Release`

### LINKED ISSUE #23623: Android release notes for `1.40.x - Release`
### Release Notes:

* Launching VPN support page will crash app if set as default #23492
* Upgrade from Chromium 103.0.5060.42 to Chromium 103.0.5060.53 #23482
* Prevent from opening Ethereum DApps dialog in inactive tab #23460
* Create providers only in secure contexts #23407
* DApps: remove beta from a Wallet item menu #23238
* Sync Nightly code causes failure on Beta and Stable #23206
* DApps: Enable DApps support by default #23159
* Provide an option to remove Top Tiles from NTP #23118
* Add pre-loaded EVMs into wallet network selector (supported by 0x) #22888
* Fix a potential bug while setting brave as default #22766
* Android and Desktop must use same code for sync QR code generation #22420
* Do not use selected network when resolving UD/ENS domains to eth address #22151
* Retire the option of using DNS over HTTPS for unstoppable domains resolution #22149
* Change unstoppable domains user opt-in interstitial from Cloudflare DoH to ethereum option #22148
* ERC721 support #19565

### PR DESCRIPTION
**Pull request by kjozwiak**
Resolves #23623

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:

N/A

### CODE REVIEW

File: CHANGELOG_ANDROID.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.40.105](https://github.com/brave/brave-browser/releases/tag/v1.40.105)
+
+ - Added Dapps support for Brave Wallet. ([#23159](https://github.com/brave/brave-browser/issues/23159))
+ - Added Non-Fungible Token support for Brave Wallet. ([#19565](https://github.com/brave/brave-browser/issues/19565))
+ - Added pre-loaded EVMs into Brave Wallet network selector (supported by 0x). ([#22888](https://github.com/brave/brave-browser/issues/22888))
```
Comment (srirambv) [inline CHANGELOG_ANDROID.md:7]: Remove added. Just pre loaded EVMs is fine

File: CHANGELOG_ANDROID.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+ - Added ability to remove "Top Tiles" from the New Tab Page via the "New Tab Page" settings. ([#23118](https://github.com/brave/brave-browser/issues/23118))
+ - Retired the option of using DNS over HTTPS for Unstoppable Domains resolution now that layer 2 support has been added. ([#22149](https://github.com/brave/brave-browser/issues/22149))
+ - Changed Ethereum Dapp providers to only be available in secure contexts. ([#23407](https://github.com/brave/brave-browser/issues/23407))
```
Comment (srirambv) [inline CHANGELOG_ANDROID.md:10]: Nit: `provider` instead of `providers`

File: CHANGELOG_ANDROID.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+ - Changed Unstoppable Domains user opt-in interstitial from Cloudflare DoH to Ethereum option. ([#22148](https://github.com/brave/brave-browser/issues/22148))
+ - Removed "Beta" from Brave Wallet via the settings menu. ([#23238](https://github.com/brave/brave-browser/issues/23238))
+ - Updated QR sync code generator to include Brave icon when creating new QR sync codes. ([#22420](https://github.com/brave/brave-browser/issues/22420)) 
+ - Fixed Ethereum Dapps dialogs being opened in inactive tabs. ([#23460](https://github.com/brave/brave-browser/issues/23460))
```
Comment (srirambv) [inline CHANGELOG_ANDROID.md:14]: Nit: `dialog` instead of `dialogs`

File: CHANGELOG_ANDROID.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.40.105](https://github.com/brave/brave-browser/releases/tag/v1.40.105)
+
+ - Added Dapps support for Brave Wallet. ([#23159](https://github.com/brave/brave-browser/issues/23159))
+ - Added Non-Fungible Token support for Brave Wallet. ([#19565](https://github.com/brave/brave-browser/issues/19565))
```
[the remark this answers is not in the document]
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: Changed to `Added support to list Non-Fungible Token (ERC721) in Brave Wallet.`

File: CHANGELOG_ANDROID.md (hunk 3 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: Fixed/Updated 👍

File: CHANGELOG_ANDROID.md (hunk 1 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md:7]: Usually use a prefix in front of each note so lets just leave it as desktop using the same one which was reviewed by several people as well.

File: CHANGELOG_ANDROID.md (hunk 2 above)
Comment (kjozwiak) [reply CHANGELOG_ANDROID.md]: Fixed/Changed

**Review by srirambv — APPROVED** [review]
++

**Pull request merged**